docs/source/deployguide/use_CA.md

```diff
@@ -68,11 +68,13 @@ Even if you don't plan to create an orderer organization, this kind of structure
 
 It is the best practice to create these folders before enrolling identities and then referencing them when issuing the enroll command through the `--mspdir` flag. Note that while the --mspdir flag is used to specify where the MSP of the CA admin is during **registration**, it is used instead during **enrollment** to specify the location on the filesystem where the folders and certificates returned by the CA will be stored.
 
-### Specifying NodeOUs
```

Comment:
I'm actually confused as to why we actually have this in the CA docs to start with.
**NodeOUs** are a Fabric construct not a Fabric CA construct.

It is true that Fabric CA will include the *role* as an *OU* in the certificate.  Do you really need to say more than that?

Reply:
The goal of the doc is to be a one stop shop for users who want to register and enroll. This is the doc we're going to point people to and tell them that they have to go through before they can create a peer or an ordering service. Once people are done here hopefully all they have to do is point to the correct folders/certs.

So I get your point, but I do think this belongs here completeness' sake.